### ISSUE TITLE
Request for Darkmode theme to be included in the Mariner WSLg distribution

### ISSUE THREAD

**Issue by EricPell** [microsoft/wslg#251]
**Is your feature request related to a problem? Please describe.**

While Windows contains themes for dark and light mode, the theme included in WSLg does not.

**Describe the solution you'd like**
 It would be nice to have a dark mode to decorate windows, perhaps what is included with the default Ubuntu Gnome Desktop.

**Describe alternatives you've considered**
As an alternative, it would be ok if we could install a theme into the Mariner package running the X11 and Wayland server, to prevent needless duplication of packages on disk, like X11, which would be installed in Ubuntu, for example. 

**Additional context**
This request originated in a discussion found here:  https://www.reddit.com/r/bashonubuntuonwindows/comments/nmsmm9/dark_mode_on_wsl2_ubuntu/

**Comment by CassianoSF** [comment]
I changed the theme with gnome-tweaks, but the theme is not applied in every app, this would be a cool enhancement, everyone knows that dark themes improve development speed up to 50% 😂

![image](https://user-images.githubusercontent.com/25011453/120085042-6a485d00-c0ab-11eb-81c9-6dd691205e2d.png)
![image](https://user-images.githubusercontent.com/25011453/120085032-543a9c80-c0ab-11eb-9c53-d446ab23ee86.png)

**Comment by hideyukn88** [comment]
Thanks for reporting, I would like to put this as dup of https://github.com/microsoft/wslg/issues/37. thanks!

**Comment by EricPell** [comment]
> Thanks for reporting, I would like to put this as dup of #37. thanks!

Does changing the Window decoration actually change the interior app color scheme? I thought it only changed the borders?

**Comment by apudiu** [comment]
After applying Yaru-dark theme it's not working for me, Checkout the borders in [Screenshot](https://prnt.sc/T1R3Ya2EW8V1)

**Comment by juandev1109** [comment]
> * gsettings set org.gnome.desktop.interface color-scheme 'prefer-dark'

This goes perfect if you execute your apps directly from terminal. If you use the links provided by W11 they'll still work in light mode.

**Comment by ninbura** [comment]
> After applying Yaru-dark theme it's not working for me, Checkout the borders in [Screenshot](https://prnt.sc/T1R3Ya2EW8V1)

Same here, launching Firefox from bash in WSL or from Windows via shortcut still results in a white border.

**Comment by MaD70** [comment]
This (adding to `/etc/environment`) worked for me where other settings failed:

> [Comment](https://www.reddit.com/r/bashonubuntuonwindows/comments/nmsmm9/comment/gzrv2gi/) by [u/ihakan123](https://www.reddit.com/user/ihakan123/) from discussion [Dark Mode on WSL2 Ubuntu](https://www.reddit.com/r/bashonubuntuonwindows/comments/nmsmm9/dark_mode_on_wsl2_ubuntu/) in [bashonubuntuonwindows](https://www.reddit.com/r/bashonubuntuonwindows/)
> <br>From another thread, people had verified that you can also just run `export GTK_THEME=Adwaita:dark` before starting your app or adding `GTK_THEME=Adwaita:dark` to your `/etc/environment` file should also give you dark mode.
